invenio_github/models.py

```diff
@@ -238,6 +238,9 @@ def enabled(self):
 
     def latest_release(self, status=None):
         """Chronologically latest published release of the repository."""
+        # Bail out fast if object not in DB session.
+        if self not in db.session:
```

Comment:
It's due to [this](https://github.com/inveniosoftware/invenio-github/blob/master/invenio_github/views/github.py#L160-L161), which creates an instance which should go in the session.

http://docs.sqlalchemy.org/en/latest/orm/session_state_management.html#session-attributes